src/redux/Registration.redux/Registration.actions.js

```diff
@@ -12,7 +12,6 @@ import {
 import { USER_TYPES } from '../User.redux/User.types';
 
 export const SET_REGISTRATION_FORM = 'SET_REGISTRATION_FORM';
-
```

Comment:
Probably shouldn't remove this whitespace... It's more clear with the whitespace that the comment block pertains to the code block below.

Reply:
You're 100% right, that was a dumb accident.